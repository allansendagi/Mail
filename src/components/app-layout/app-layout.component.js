import React from 'react';
import './app-layout.styles.scss';
import HeaderNav from '../../containers/headernav/headernav.container';

export function AppLayout(props) {
	return (
		<div className='app-layout'> 
			<HeaderNav/>
			{props.children}
		</div>
	)
}