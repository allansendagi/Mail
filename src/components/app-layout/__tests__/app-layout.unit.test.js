import React from 'react';
import {shallow, configure} from 'enzyme';
import {AppLayout} from '../app-layout.component';
import Adapter from 'enzyme-adapter-react-16';


configure({adapter: new Adapter()});
test('renders empty <AppLayout/>', () => {
  const wrapper = shallow(
    <AppLayout/>
  );
  expect(wrapper).toMatchSnapshot();
});

test('renders <AppLayout/> with one child', () => {
  const wrapper = shallow(
    <AppLayout>
      <div>Child 1</div>
    </AppLayout>
  );
  expect(wrapper).toMatchSnapshot();
});

test('renders <AppLayout/> with children', () => {
  const wrapper = shallow(
    <AppLayout>
      <div>Child</div>
      <div>
        <span>Child</span>
        <p>Child</p>
      </div>
    </AppLayout>
  );
  expect(wrapper).toMatchSnapshot();
});