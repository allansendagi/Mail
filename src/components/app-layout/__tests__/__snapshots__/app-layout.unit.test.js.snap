// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <AppLayout/> with children 1`] = `ShallowWrapper {}`;

exports[`renders <AppLayout/> with one child 1`] = `ShallowWrapper {}`;

exports[`renders empty <AppLayout/> 1`] = `ShallowWrapper {}`;
